src/regions/VideoRectangleRegion.js

```diff
@@ -4,7 +4,8 @@ import NormalizationMixin from '../mixins/Normalization';
 import RegionsMixin from '../mixins/Regions';
 import Registry from '../core/Registry';
 import { AreaMixin } from '../mixins/AreaMixin';
-import { interpolateProp, onlyProps, VideoRegion } from './VideoRegion';
+import { onlyProps, VideoRegion } from './VideoRegion';
+import { interpolateProp } from '../utils/props';
```

Comment:
not sure if common utils is the right place for it. interpolation only needed for video as we have "animation" there

Reply:
We have the same common utils like `selection-tools` or `magic-wand`. And it still can be reused later for animations of something. It's edge case but I prefer to take it in account.

Reply:
makes sense, but we should be careful about putting stuff in common utils anyways. let's keep it for now though